cypress/e2e/smoke/balances.cy.js

```diff
@@ -0,0 +1,228 @@
+        .should('be.visible')
+
+      // Balance should contain WETH
+      cy.contains('Wrapped Ether').parents('tr').find('td').eq(TOKEN_AMOUNT_COLUMN).contains('WETH')
+    })
+
+    it('should have USD Coin', () => {
+      // Row should have an image with alt text "USD Coin"
+      cy.contains('USD Coin')
+        .parents('tr')
+        .within(() => {
+          cy.get('img[alt="USDC"]').should('be.visible')
+        })
+
+      // Asset name column contains link to block explorer
+      cy.contains('USD Coin')
+        .parents('tr')
+        .find('td')
+        .eq(ASSET_NAME_COLUMN)
+        .get('a[aria-label="View on rinkeby.etherscan.io"]')
+        .should('be.visible')
+
+      // Balance should contain USDT
+      cy.contains('USD Coin').parents('tr').find('td').eq(TOKEN_AMOUNT_COLUMN).contains('USDC')
+    })
+  })
+
+  describe('values should be formatted as per locale', () => {
+    it('should have Token and Fiat balances formated as per specification', () => {
+      cy.contains('Dai')
+        .parents('tr')
+        .within(() => {
+          cy.get('td').eq(TOKEN_AMOUNT_COLUMN).contains('803.292M DAI')
+          cy.get('td').eq(FIAT_AMOUNT_COLUMN).contains(fiatRegex)
+        })
+
+      cy.contains('Wrapped Ether')
+        .parents('tr')
+        .within(() => {
+          cy.get('td').eq(TOKEN_AMOUNT_COLUMN).contains('0.96788 WETH')
+          cy.get('td').eq(FIAT_AMOUNT_COLUMN).contains(fiatRegex)
+        })
+
+      // Strict match because other tokens contain "Ether" in their name
+      cy.contains(/^Ether$/)
+        .parents('tr')
+        .within(() => {
+          cy.get('td').eq(TOKEN_AMOUNT_COLUMN).contains('0.3 ETH')
+          cy.get('td').eq(FIAT_AMOUNT_COLUMN).contains(fiatRegex)
+        })
+
+      cy.contains('Uniswap')
+        .parents('tr')
+        .within(() => {
+          cy.get('td').eq(TOKEN_AMOUNT_COLUMN).contains('0.01828 UNI')
+          cy.get('td').eq(FIAT_AMOUNT_COLUMN).contains(fiatRegex)
+        })
+
+      cy.contains('USD Coin')
+        .parents('tr')
+        .within(() => {
+          cy.get('td').eq(TOKEN_AMOUNT_COLUMN).contains('13,636,504 USDC')
+          cy.get('td').eq(FIAT_AMOUNT_COLUMN).contains(fiatRegex)
+        })
+
+      cy.contains('Gnosis')
+        .parents('tr')
+        .within(() => {
+          cy.get('td').eq(TOKEN_AMOUNT_COLUMN).contains('< 0.00001 GNO')
+          cy.get('td').eq(FIAT_AMOUNT_COLUMN).contains(fiatRegex)
+        })
+
+      cy.contains(/^0x$/)
+        .parents('tr')
+        .within(() => {
+          cy.get('td').eq(TOKEN_AMOUNT_COLUMN).contains('1.003 ZRX')
+          cy.get('td').eq(FIAT_AMOUNT_COLUMN).contains(fiatRegex)
+        })
+    })
+  })
+
+  describe('fiat currency can be changed', () => {
+    it('should have ETH as default currency', () => {
+      // First row Fiat balance should not contain EUR
+      cy.get(balanceSingleRow).first().find('td').eq(FIAT_AMOUNT_COLUMN).should('not.contain', '€')
```

Comment:
```suggestion
      cy.get(balanceSingleRow).first().find('td').eq(FIAT_AMOUNT_COLUMN).should('not.contain', 'EUR')
```

Reply:
A PR was merged today with this change

Reply:
So is no longer "€" and "$"? currently if I put USD and EUR it will fail